devito/at_controller.py

```diff
@@ -0,0 +1,344 @@
+import os
+import stat
+import math
+import shutil
+import cpuinfo
+import subprocess
+
+# global vars.
+# report folder will be created in current dir. Change if necessary
+reports_folder_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "auto-tune-reports")
```

Comment:
Yes. Configuration should not involve changes deep in the code. Env variable or config file is an open question for now. It could be provided as a constructor parameter for the At controller.